smallfile_wrapper/trigger_smallfile.py

```diff
@@ -137,6 +141,23 @@ def emit_actions(self):
                             interval['99%'] = float(flds[10])
                             yield interval, '-rsptimes'
 
+            if self.clients > 1:
+                self.logger.info("Number of smallfile pods > 1, synchronizing them for the next operation")
+                r = redis.StrictRedis(self.redis_host, 6379)
+                p = r.pubsub()
+                p.subscribe("smallfile")
+                subscribers = int(r.pubsub_numsub("smallfile")[0][1])
+                if subscribers == self.clients:
```

Comment:
Do we want to add a limit on the number of times we check to see if the clients are synced? For example if a client pod goes into OOMKill or some other undesirable fault, should we exit notifying the user there is a problem?

Reply:
I've added a redis timeout to the implementation which will make redis subscription to timeout if no message is received on that channel. By default, 60 seconds. Can be  overridden by an env. var